socialhome/content/tests/test_serializers.py

```diff
@@ -89,3 +89,15 @@ def test_user_has_shared_true_if_shared(self):
         self.content.share(self.profile)
         serializer = ContentSerializer(context={"request": Mock(user=self.user)})
         self.assertTrue(serializer.get_user_has_shared(self.content))
+
+    def test_tags_if_no_tag(self):
+        self.content.tags.clear()
+        serializer = ContentSerializer(context={"request": Mock(user=self.user)})
+        self.assertEquals(serializer.get_tags(self.content), [])
```

Comment:
These tests should now check the serialized content object and what `tags` list contains. You will find the list as `serializer.data["tags"]`.